renderdoc/driver/vulkan/vk_serialise.cpp

```diff
@@ -876,11 +890,6 @@ SERIALISE_VK_HANDLES();
   PNEXT_UNSUPPORTED(VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_IMAGE_VIEW_IMAGE_FORMAT_INFO_EXT)                \
   PNEXT_UNSUPPORTED(VK_STRUCTURE_TYPE_FILTER_CUBIC_IMAGE_VIEW_IMAGE_FORMAT_PROPERTIES_EXT)             \
                                                                                                        \
-  /* VK_EXT_full_screen_exclusive */                                                                   \
```

Comment:
The `VK_EXT_hdr_metadata` structure is't implemented at all and is still listed as `PNEXT_UNSUPPORTED`.

(Here since I can't comment on lines outside of diff context).